### DISCUSSION TITLE
Menu 组件的官方说明文档上 MenuItemGroupType 对 label 的类型描述为 ReactNode，应该是 VueNode。

### DISCUSSION THREAD

**Discussion by nd-lqj** [vueComponent/ant-design-vue#7075] [General]
这个类型说明是从 react 版本复制过来的吗？

**Comment by nd-lqj** [comment]
![image](https://github.com/vueComponent/ant-design-vue/assets/9653180/af6d24eb-5533-4a2d-82b1-c923684dfd75)

**Comment by webvs2** [comment]
@nd-lqj  The website seems to have corrected the type
